### PR TITLE
remove 4010 standard

### PR DESCRIPTION
**Pull request by aethelwulffe**
snuffing 4010 code from gen837

### CODE REVIEW

**Comment by aethelwulffe** [comment]
Quite willing to resolve any conflicts after Liberation.

File: library/gen_x12_837.inc.php [hunk 1]
```diff
@@ -440,11 +367,12 @@ function gen_x12_837($pid, $encounter, &$log, $encounter_claim=false) {
   //   $log .= "*** CMS ID is missing for payer '" . $claim->payerName() . "'.\n";
   // }
 
-  if (true) { // !$CMS_5010
+  if (true) {
```
Comment (tmccormi) [inline library/gen_x12_837.inc.php]: Any good reason to leave this non if statement in?

**Comment by tmccormi** [comment]
First code review via cellphone

File: library/gen_x12_837.inc.php (hunk 1 above)
Comment (aethelwulffe) [inline library/gen_x12_837.inc.php]: As it does nothing at all?  Well shouldn't we keep a little useless legacy code in?  Ain't that tradition?
Making changes now.

Comment (aethelwulffe) [inline library/gen_x12_837.inc.php]: It is a block placeholder for a piece we really should revisit.  Maybe I should just wrap it in a todo set of comment lines.

**Comment by aethelwulffe** [comment]
Changed to include a review comment.  This section will be attended to in a full re-write of the gen and parse scripts.

**Pull request merged**